src/main/java/com/cleanroommc/groovyscript/compat/vanilla/ItemStackMixinExpansion.java

```diff
@@ -220,6 +223,16 @@ default ItemStack withDamage(int meta) {
         return withMeta(meta);
     }
 
+    default ItemStack destroy() {
```

Comment:
should this check if the item is already empty? the other place i see this being done does so.